### ISSUE TITLE
Incorrect documentation for {{{raw}}} helper

### ISSUE THREAD

**Issue by ao-rix** [sunng87/handlebars-rust#254]
Looks like documentation for `{{{{raw}}}}` helper is wrong. From the current docs (https://docs.rs/handlebars/1.2.0/handlebars/):
```
{{{{#raw}}}} ... {{{{/raw}}}} escape handlebars expression within the block
```

From #68 test:
```
let t = Template::compile("a{{{{raw}}}}{{content}}{{else}}hello{{{{/raw}}}}".to_string()).ok().unwrap();
```

`{{{{#raw}}}}` vs `{{{{raw}}}}`, excess hash sign.

**Comment by sunng87** [comment]
I'm sorry the doc was obsolete. I will fix it.

**Comment by sunng87** [comment]
Fixed in b9bb1fdf0a77f91813fa51612dbdbb4871df9915
